source/tests/pt/test_stat.py

```diff
@@ -170,20 +170,15 @@ def test_descriptor(self):
                 if key in sys.keys():
                     sys[key] = sys[key].to(env.DEVICE)
         stat_dict = my_en.compute_input_stats(sampled)
-        my_en.init_desc_stat(**stat_dict)
         my_en.mean = my_en.mean
         my_en.stddev = my_en.stddev
-        self.assertTrue(
-            np.allclose(
-                self.dp_d.davg.reshape([-1]), my_en.mean.cpu().reshape([-1]), rtol=0.01
-            )
+        np.testing.assert_allclose(
+            self.dp_d.davg.reshape([-1]), my_en.mean.cpu().reshape([-1]), rtol=0.01
         )
-        self.assertTrue(
-            np.allclose(
-                self.dp_d.dstd.reshape([-1]),
-                my_en.stddev.cpu().reshape([-1]),
-                rtol=0.01,
-            )
+        np.testing.assert_allclose(
+            self.dp_d.dstd.reshape([-1]),
+            my_en.stddev.cpu().reshape([-1]),
+            rtol=0.01,
```

Comment:
not sure if the rtol is too loose.

Reply:
Absulotely! When I increased rtol, I found a bug in the code. The original tol is too low to find the bug.

Reply:
I found there are another issue...

Reply:
I found coordinates fed to TF (line 150) is different from those fed to PT (line 171).

Need help here...

Reply:
I make it work in a hack way: picking only 2 frames in to a new data directory.

Reply:
> I make it work in a hack way: picking only 2 frames in to a new data directory.

confused by the hacking. If i understand correctly, the statistics should be the same if datasets and numpy random seeds are the same.

Reply:
> confused by the hacking. If i understand correctly, the statistics should be the same if datasets and numpy random seeds are the same.

It looks like TF and PT don't have consistent random behavior. It may need @CaRoLZhangxy to resolve.

Reply:
could you please raise an issue?